server/server.js

```diff
@@ -2,11 +2,12 @@ var express = require('express');
 var app = express();
 var port = 8000;
 var clearDatabase = (process.env.NODE_ENV !== 'production' && process.env.CLEAR_DATABASE === 'true')
+var server = require('http').createServer(app);
```

Comment:
express creates the server for you when you use the listen function. check out the documentation for app.listen : http://expressjs.com/api.html